apps/website/src/components/Playground/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { SandpackProvider, SandpackLayout, SandpackCodeEditor, SandpackPreview } from '@codesandbox/sandpack-react';
+import { useColorMode } from '@docusaurus/theme-common';
+import AppCode from '!!raw-loader!./template/App.js';
+import StylesCode from '!!raw-loader!./template/Styles.js';
+
+const PLAYGROUND_HEIGHT = 400;
+
+export default function Playground() {
+  const { isDarkTheme } = useColorMode();
+  const sandpackTheme = isDarkTheme ? 'dark' : 'github-light';
+  return (
+    <SandpackProvider
+      template="react"
+      customSetup={{
+        dependencies: { '@griffel/core': 'latest', 'highlight.js': 'latest', 'js-beautify': 'latest' },
+        files: { '/App.js': { code: AppCode, hidden: true }, '/Styles.js': { code: StylesCode, active: true } },
+      }}
+    >
+      <SandpackLayout theme={sandpackTheme}>
+        <SandpackCodeEditor showLineNumbers customStyle={{ height: PLAYGROUND_HEIGHT }} />
```

Comment:
unfortunately without the `editorHeight`  option of the default `<Sandpack />` component, it's really the only way to make the playground bigger, editing the layout component above does not do anything

Reply:
For now IMO it's ok. In followup PRs it would nice to introduce autosizing/resizing feature.